src/config/kubeconfig.js

```diff
@@ -10,15 +10,28 @@ function path() {
   if (process.env.KUBECONFIG && process.env.KUBECONFIG.length > 0) {
     const files = process.env.KUBECONFIG.split(pth.delimiter);
     // Only returning the path to the first file if there are multiple.
-    return files[0];
+    if (files.length) {
+      return files[0];
```

Comment:
We're probably doing the wrong API if we support multiple files (we won't show some of the entries).  But that's probably a different PR.
This does not work correctly if the file file in `KUBECONFIG` doesn't exist.

Reply:
It turns out that if the first file in `process.env.KUBECONFIG` doesn't exist, you get an exception from `node_modules/@kubernetes/client-node/dist/config.d.ts` when it tries to load the default config from the first file in process.env.KUBECONFIG.  So I moved this filtering code to that module. Not completely sure we should even bother doing that -- if someone wants to do something like this:

```
KUBECONFIG=$HOME/morris/cat:$HOME/mason/reese:$HOME/.kube/not-a-config npm run dev
```

should we really get in their way?

Reply:
I was thinking about the case where multiple _valid_ files are listed, but that really doesn't matter since the same issue existed before the PR. I'll just resolve this.